cluster_orion_hunt.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Probably should move all those cluster scripts into a ./cluster directory to kept root clean.